### PR TITLE
[ISSUE-2656] Update GZIP compression type in user doc

### PR DESCRIPTION
**Pull request by yuqi1129**
## Description
see https://github.com/apache/iotdb/issues/2656

### CODE REVIEW

**Comment by wangchao316** [comment]
+1

File: docs/zh/UserGuide/Concept/Compression.md [hunk 1]
```diff
@@ -28,5 +28,6 @@ IoTDB允许在创建一个时间序列的时候指定该列的压缩方式。现
 * UNCOMPRESSED（不压缩）
 * SNAPPY压缩
 * LZ4压缩
+* GZIP压缩
```
Comment (HTHou) [inline docs/zh/UserGuide/Concept/Compression.md:31]: Please also update the line 26.
We have 4 compression types now. :)

File: docs/UserGuide/Concept/Compression.md [hunk 2]
```diff
@@ -31,4 +31,6 @@ IoTDB allows you to specify the compression method of the column when creating a
 
 * LZ4
 
+* GZIP
```
Comment (HTHou) [inline docs/UserGuide/Concept/Compression.md:34]: Same. Update the line 26?

File: docs/zh/UserGuide/Concept/Compression.md (hunk 1 above)
Comment (yuqi1129) [reply docs/zh/UserGuide/Concept/Compression.md:31]: Sure

Comment (qiaojialin) [reply docs/zh/UserGuide/Concept/Compression.md:31]: 改成 "有以下几种" 吧。。

**Review by HTHou — APPROVED** [review]

**Pull request merged**